atomictests.py

```diff
@@ -0,0 +1,133 @@
+import unittest
+import re
+from .testutils import system, if_atomic
+
+
+@unittest.skipUnless(if_atomic(), "It's not an Atomic image")
+class TestDockerStorageSetup(unittest.TestCase):
+
+    def test_journalctl_logs(self):
```

Comment:
Might be good to include some "known" good output here so anyone reading can get an idea of what the output should look like. This is good for when tests fail and someone tries to go in to see what the test was doing.